SEM_2_PRACTICALS/prac_06/guitar_test.py

```diff
@@ -0,0 +1,16 @@
+from SEM_2_PRACTICALS.prac_06.guitar import Guitar
```

Comment:
See this name, "SEM_2..."? It doesn't follow the Python convention for module naming. That's why we prescribed proper names way back at the start: https://github.com/CP1404/Practicals/tree/master/prac_01#project-structure-use-one-project-for-all-practicals
Please get used to following instructions. Everything matters.